Source/WebCore/rendering/svg/SVGInlineTextBox.h

```diff
@@ -23,14 +24,21 @@
 
 #include "LegacyInlineTextBox.h"
 #include "LegacyRenderSVGResource.h"
+#include "RenderSVGResourcePaintServer.h"
 #include "SVGTextFragment.h"
 
 namespace WebCore {
 
 class LegacyRenderSVGResource;
 class RenderSVGInlineText;
+class SVGPaintServerHandling;
 class SVGRootInlineBox;
 
+#if ENABLE(LAYER_BASED_SVG_ENGINE)
```

Comment:
I think you can forward reference just about any class, so I don't think this part has to be protected.

Reply:
Wasn't sure if we had a preferred style regarding that topic. Remove the guard.